README.md

```diff
@@ -2,7 +2,7 @@
 
 #  School Library
 
-> OOP school library in Ruby. Practicing classes.
+> OOP school library in Ruby. Practicing classes. Creating simple UI for a school library. 
 
```

Comment:
- [ ] (Optional) It would be nice to include the instruction for running the console after this repository is cloned. For example, `run ruby main.rb to start the console`. 👍

Reply:
I will add it. Thank you for your good review. I appreciated it a lot. 💯

Reply:
Hi @saied2035, you are welcome